homeassistant/components/switcher_kis/switch.py

```diff
@@ -0,0 +1,294 @@
+"""Home Assistant Switcher Component.
+
+For controlling the Switcher Boiler Device (https://www.switcher.co.il/).
+Please follow configuring instructions here:
+    https://www.home-assistant.io/components/switcher_kis/
+
+Author: Tomer Figenblat
+
+This cannot be configured as a sensor platform,
+Please follow the instruction of configuring the switcher_kis component.
+"""
+
+from asyncio import TimeoutError as AsyncioTimeoutError
+from asyncio import wait_for
+from datetime import datetime
+from functools import partial
+from logging import getLogger
+from typing import Any, Awaitable, Callable, Dict, Optional
+
+from aioswitcher.consts import COMMAND_OFF, COMMAND_ON
```

Comment:
Dependencies can only be imported inside functions.

Reply:
My bad.
I was under the impression that because the installation of the library was called by the component itself, then it will be safe for the platforms to import outside of the functions, seeing it's already been installed.
I will correct this and any resembling change requests.